js/components/moter/motebehov/SvarMotebehovSkjema.js

```diff
@@ -0,0 +1,239 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Hovedknapp } from 'nav-frontend-knapper';
+import { connect } from 'react-redux';
+import { Field, reduxForm, getFormValues } from 'redux-form';
+import { Link } from 'react-router';
+import {
+    proptypes as motebehovProptypes,
+    svarMotebehovSender,
+} from 'moter-npm';
+import Tekstomraade from '../../skjema/Tekstomraade';
+import Radioknapper from '../../skjema/Radioknapper';
+
+export const tekstfeltRegex = new RegExp('.*<[^ ][^>]+[^ ]>.*');
+
+export const felterPt = PropTypes.shape({});
+
+const SVAR_MOTEBEHOV_SKJEMANAVN = 'svarMotebehov';
```

Comment:
Denne kan evt legges til `enums/skjemanavn.js`

Reply:
Er tanken da å ha et sentralisert sted for "id" til skjema dersom man ønsker å hente state inn i et skjema fra et annet?